src/components/ProfilePage/LevelBox/LevelBox.tsx

```diff
@@ -0,0 +1,22 @@
+import { Skeleton, useTheme } from '@mui/material';
+import { ProfilePageTypography } from '../ProfilePage.style';
+import { TierBadgeBox } from './TierBox.style';
+
+interface LevelBoxProps {
+  level?: number;
+  loading: boolean;
+}
+
+export const LevelBox = ({ level, loading }: LevelBoxProps) => {
+  return (
+    <TierBadgeBox>
+      {loading ? (
+        <Skeleton variant="text" sx={{ fontSize: '42px', width: '132px' }} />
+      ) : (
+        <ProfilePageTypography fontSize={'18px'} lineHeight={'24px'}>
+          {`LEVEL ${level}`}
```

Comment:
Do we want to localize that string?

Reply:
No, it is the LEVEL inside your loyalty, the marketing team will speak about "LEVEL" inside the "Loyalty Pass". As for as I know we don't localize "loyalty pass" so should not "level" as well